### ISSUE TITLE
Nexus tree file reader dies with empty internal lines

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#80]
e.g.
```
begin TREES;

	TRANSLATE
...
```

**Comment by josephwb** [comment]
Fixed translation example above with 1e58dd0abb085b5bb37edf881c23de7c6aa1df33.

Still have a problem if there is a space between the last tree and `End`:
```
...
tree foo = ((A,B),C);

End;
```

**Comment by josephwb** [comment]
And 2d4c4f2260fb642026c2c8510c343c3429cc49c0 takes care of other internal 'empty' lines (may have spaces/tabs but no data).

Might be worth it to check out newick reader. But do we _really_ want to support this stuff?
